skimage/transform/_radon_transform.pyx

```diff
@@ -172,10 +173,10 @@ cpdef bilinear_ray_update(cnp.double_t[:, :] image,
 
 
 @cython.boundscheck(True)
-def sart_projection_update(cnp.double_t[:, :] image not None,
-                           cnp.double_t theta,
-                           cnp.double_t[:] projection not None,
-                           cnp.double_t projection_shift=0.):
```

Comment:
Any clue why bounds check is enabled here?

Reply:
Not really, I was also wondering why... I will investigate a little bit on this... Thank you anyway @hmaarrfk for your review.